csharp/device/doc/devbox_setup.md

```diff
@@ -1,16 +1,19 @@
 # Prepare your development environment
 
-This document describes how to prepare your development environment to use the *Microsoft Azure IoT device SDK for C#*.
+This document describes how to prepare your development environment to build and use the **Microsoft .NET Device SDK for Azure IoT Devices**
```

Comment:
We need to be consistent across the document (and across the different languages) with the name for the client SDK. I suggest we align to:
Azure IoT device client SDK for C#